mlir_graphblas/src/lib/GraphBLAS/GraphBLASLowerPass.cpp

```diff
@@ -3255,6 +3255,39 @@ class LowerCommentRewrite : public OpRewritePattern<graphblas::CommentOp> {
   };
 };
 
+class LowerPrintRewrite : public OpRewritePattern<graphblas::PrintOp> {
+public:
+  using OpRewritePattern<graphblas::PrintOp>::OpRewritePattern;
+  LogicalResult matchAndRewrite(graphblas::PrintOp op,
+                                PatternRewriter &rewriter) const override {
+
+    ModuleOp module = op->getParentOfType<ModuleOp>();
+    Location loc = op->getLoc();
+
+    for (auto enumerated_pair : llvm::enumerate(llvm::zip_longest(op.strings(), op.values()))) {
+      auto pair = enumerated_pair.value();
+      Optional<Attribute> stringAttribute = std::get<0>(pair);
+      Optional<Value> val = std::get<1>(pair);
+
+      if (stringAttribute) {
+        StringRef currentString = stringAttribute.getValue().dyn_cast<StringAttr>().getValue();
+        callPrintString(rewriter, module, loc, currentString);
+      } else if (enumerated_pair.index() != 0) 
+	callPrintString(rewriter, module, loc, " ");
+
+      if (val)
+        callPrintValue(rewriter, module, loc, val.getValue());
+      else 
+	callPrintString(rewriter, module, loc, " ");
```

Comment:
Looks like you used a tab instead of 8 spaces for these `callPrintString` lines. Should be fixed.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/dcb0853b4ed00df111b0bac93f024c49407f05b9 fixes this.